cost/aws/schedule_instance/README.md

```diff
@@ -67,14 +69,18 @@ This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Auto
   - `ec2:DescribeInstances`
```

Comment:
I can't select it from here (and I know this is not directly related to the change you are making), but the content on Line 66 should be under the Prerequisites section but before the Credential configuration section

e.g., https://github.com/flexera-public/policy_templates/tree/master/cost/aws/rightsize_ec2_instances